synthtool/gcp/templates/python_library/CONTRIBUTING.rst

```diff
@@ -22,7 +22,7 @@ In order to add a feature:
   documentation.
 
 - The feature must work fully on the following CPython versions:  2.7,
```

Comment:
Could you drop 2.7 as well? I know a handful of handwrittens still *technically* work with 2.7 but otherwise it is gone.

Reply:
36637b5